Feature/crud operation with security

AlbertNadoyan opened the pull request.

AlbertNadoyan committed:
SpringBoot CRUD operation.

AlbertNadoyan committed:
USER with security.

pdasilem commented on src/main/java/com/example/springboothibernate/service/impl/UserServiceImpl.java:
```diff
@@ -0,0 +1,55 @@
+
+    @Override
+    public User updateUser(Long id, User user) {
+        User userById = userRepository.findById(id).orElseThrow(() -> new RuntimeException("User not found"));
```
не красиво это - кидать в ексепшн если ты не используешь хэндлеры.

AlbertNadoyan replied:
I fixed the problem.

AlbertNadoyan committed:
I fixed the problem
Created a custom exception`UserNotFound and used ExceptionController and handled by exception type.
